src/makecolors.js

```diff
@@ -1,37 +1,31 @@
 /* eslint-disable indent */
 /* eslint-disable no-console */
-const fs = require('fs');
-const crypto = require('crypto');
+const fs = require('node:fs');
+const { promises: fsPromises } = require('node:fs');
+const crypto = require('node:crypto');
 const colors = require('../frontend/src/colors');
 
-function generateHashes() {
-  const scssFileBuffer = fs.readFileSync('./frontend/src/colors.scss');
-  const scsshash = crypto.createHash('sha256');
-  scsshash.update(scssFileBuffer);
-
-  const scsshex = scsshash.digest('hex');
-  fs.writeFile('./colorsscsschecksum', scsshex, (err) => {
-    if (err) {
-      console.error('cannot generate colors.scss hash');
-      return;
-    }
+async function generateHashes() {
+  try {
+    const scssFileBuffer = await fs.promises.readFile('./frontend/src/colors.scss');
```

Comment:
It looks like you split out fs.promises above as fsPromises but then didn't use it? I don't really care which approach you take here, just noting a potential point for clean up.